target/linux/ipq40xx/image/generic.mk

```diff
@@ -841,6 +841,22 @@ define Device/netgear_orbi
 	DEVICE_PACKAGES := ath10k-firmware-qca9984-ct e2fsprogs kmod-fs-ext4 losetup
 endef
 
+define Device/netgear_lbr20
+	$(call Device/netgear_orbi)
+	DEVICE_MODEL := LBR20
+	NETGEAR_BOARD_ID := LBR20
+	NETGEAR_HW_ID := 29766182+0+256+512+2x2+2x2+2x2+1
+	KERNEL_SIZE := 3932160
```

Comment:
`3840k`, also is there a reason for such a small kernel?
Does bootloader support reading kernel from UBI?

Reply:
Ok, it's weird if they disabled UBI support as that was enabled by default by QCA.

3840k is rather small and you will have kernel size issues rather soon.
Does it at least support LZMA compressed kernels?

Reply:
So just to be on safe side increased kernel size partition to 0x500000 (5 242 880 b).
No problems with comming back to orginal firmware , NMPRFlash is replacing it.